docs/AccessControl.md

```diff
@@ -0,0 +1,168 @@
+by manually adding keys on both servers.
+
+Nagios access is managed via the `/usr/local/nagios/etc/htpasswd.users` file
+(`.htaccess` format) on the server. Shell access is currently manually
+controlled, but will be moved to a Bastillion group.
+
+**Onboarding (Normal user)**: Add to infrastructure-secret for AWX access,
+and Nagios htpasswd.users file.  Shell access to Nagios, TRSS, and AWX can be
+granted if required.
+**Offboarding (Normal user)**: Reverse the above
+
+**Onboarding (Superadmin user)**: Add to `AdoptOpenJDK*jenkins-admins` GitHub team
+and grant shell access via root's authorized_keys to the jenkins server,
+and Bastillion. Note that Bastillion admin access is available through
+BitWarden in the `Internal Services` group.
+
+**Offboarding (Superadmin user)**: Remove from the above, change Bastillion
+admin password
+
+## Bastillion
+
+Since I've mentioned Bastillion in the earlier section, here are the full
+list of groups which we have defined on the server and what they control:
+
+Note that while Bastillion typically controls access to the root account,
+there is also a `jenkins` group which the infrastructure team is added to
+which allows direct access to the jenkins user. This can, if required, be
+granted to people outside the infrastructure team.
+
+Group name | What is in it | Who has access
+--- | --- | ---
+jenkins | jenkins user on all systems | infra team
+root-marist | All marist systems |  infra team + Marist keys
+root-aix | Root access to AIX hosts | Infra team + AIX tools + NIM
+root-equinix | Root access to Equinix hosts | Infra team + Equinix keys
+root | Root access to all other build+test hosts | Infra team
+infrahosts | root access to infra services | Infra team
+infraadmin | root access to Jenkins/Bastillion | TBC 
+root-build | root access to static build machines | (Not yet implemented 
+
+## Temporary test system access
+
+Occasionally non-infrastructure team members require access to the machines
+in order to debug problems when they are not able to do so on their local
+machines, for example if it's a type of machine they do not otherwise have
+access to. In this case we grant people the least amount of access possible.
+
+To request this, a user should create an issue in the infrastructure
+repository using [this
+template](https://github.com/adoptium/infrastructure/issues/new?assignees=sxa&labels=Temp+Infra+Access&template=machineaccess.md&title=Access+request+for+%3Cyour+username%3E)
+which will be labelled as "Temp Infra Access" and allows them to specify and
+justify the level of access.  In most cases this will be a normal user
+account which will be granted access via their ssh key.  In very rare cases
+access to jenkins or root may be provided, although bear in mind this will
+be very short lived as Bastillion will typically override that access.  In
+either case, the issue should be left open until the access is removed.
```

Comment:
I feel duty-bound to mention granting time-limited ssh access control again by key signing like the early days of AOJDK. I know I always say that! One day.

Reply:
I actually have no recollection of that - do you have a reference? Feel free to raise an issue on it as I'm not sure how that was implemented.

Are you happy to approve this with the other changes? For some reason I'm not getting a "Re-request review" option for you on this.